tools/topology/topology2/production/tplg-targets-ace3.cmake

```diff
@@ -3,6 +3,11 @@
 # Array of "input-file-name;output-file-name;comma separated pre-processor variables"
 list(APPEND TPLGS
 # SDW topology for PTL RVP
+"cavs-sdw\;sof-ptl-rt721\;PLATFORM=ptl,SDW_DMIC=1,NUM_SDW_AMP_LINKS=1,\
+SDW_AMP_FEEDBACK=false,SDW_SPK_STREAM=Playback-SmartAmp,SDW_DMIC_STREAM=Capture-SmartMic,\
+SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack,\
+HDMI1_ID=7,HDMI2_ID=8,HDMI3_ID=9"
```

Comment:
Does it assume PCH DMIC is used? So that your HDMI IDs are 7, 8, and 9. If so, you have to append `-4ch` or `-2ch` to the topology name.

Reply:
updated.. per suggestion.
thanks

Reply:
Thanks, have updated accordingly.